cli/src/pcluster/config/cluster_config.py

```diff
@@ -311,6 +312,19 @@ def _register_validators(self, context: ValidatorContext = None):
         self._register_validator(EbsVolumeSizeSnapshotValidator, snapshot_id=self.snapshot_id, volume_size=self.size)
         self._register_validator(DeletionPolicyValidator, deletion_policy=self.deletion_policy, name=self.name)
 
+    @property
+    def is_managed(self):
+        """Return True if the volume is managed."""
+        return self.volume_id is None
+
+    @property
+    def availability_zone(self):
+        """Return the availability zone of an existing EBS volume."""
+        if not self.is_managed:
+            return AWSApi.instance().ec2.describe_volume(self.volume_id)["AvailabilityZone"]
```

Comment:
nit - if output of the describe is cached, this can be simplified into
```
return AWSApi.instance().ec2.describe_volume(self.volume_id).get("AvailabilityZone","")
```